lib/tasks/fake_data.rake

```diff
@@ -17,15 +18,29 @@ class FakeDataGenerator
         password_confirmation: password,
         username: Faker::Internet.user_name(specifier: name, separators: %w(_)),
         created_at: User::NEW_USER_DAYS.days.ago,
+        karma: Random.rand(User::MIN_KARMA_TO_FLAG * 2),
+        about: Faker::Lorem.sentence(word_count: 7),
+        homepage: Faker::Internet.url,
+        invited_by_user: User.select(&:can_invite?).sample,
       }
-      create_args.merge!(is_moderator: true) if i % 7 == 0
+      create_args.merge!(is_admin: true) if i % 8 == 0
       users << User.create!(create_args)
+      if i % 7 == 0
+        users[i].grant_moderatorship_by_user!(mod)
+      end
+      if i % 3 == 0
+        users[i].enable_invite_by_user!(mod)
```

Comment:
What effect does this have? By default the new users all have the ability to invite (well, after `NEW_USER_DAYS`). I think this would only affect those with invites disabled, which you do a few lines down?

Reply:
I did this because the accounts that are created should still appear as new since they won't be NEW_USER_DAYS old yet. Therefore `invited_by_user: User.select(&:can_invite?).sample` will would be none of the new accounts and I wanted to have a more realistic looking user tree. 

I was disabling some accounts then to generate good moderation data.

Reply:
As expected, you were right and this wasn't needed. :smile: